### PR TITLE
PR: Put section header rows in the file switcher at item level

### PR DESCRIPTION
**Pull request by juanis2112**
## Description of Changes

* [ ] Wrote at least one-line docstrings (for any new functions)
* [ ] Added unit test(s) covering the changes (if testable)
* [x] Included a screenshot (if affecting the UI)

Remove section header rows in the file switcher and add them to the left of the file name in blue.

Screen Shot 2019-05-02 at 6 08 59 PM

### Issue(s) Resolved

Fixes Item 6 of Issue #8580 

### Affirmation

By submitting this Pull Request or typing my (user)name below,
I affirm the [Developer Certificate of Origin](https://developercertificate.org)
with respect to all commits and content included in this PR,
and understand I am releasing the same under Spyder's MIT (Expat) license.

I certify the above statement is true and correct:
Juanis2112

### CODE REVIEW

**Comment by ccordoba12** [comment]
@juanis2112, please update your screenshot above to let's see your latest changes.

Also, please merge with master to get the fix I pushed a couple of days ago for Python 3.5/CircleCI.

**Comment by CAM-Gerlach** [comment]
Besides ``Editor``, what other possible section headers are there?

File: spyder/widgets/fileswitcher.py [hunk 1]
```diff
@@ -669,17 +670,24 @@ def setup_file_list(self, filter_text, current_path):
...
                                                     short_paths[index])
                 else:
-                    text_item += (u" &nbsp; <span style='color:{0:};"
+                    text_item += (u" &nbsp; <span style='float: left; color:{0:};"
                                   "font-size:{1:}pt'>{2:}"
                                   "</span>").format(self.PATH_FG_COLOR,
                                                     path_text_font_size,
                                                     paths[index])
+                    title = self.widgets[index][1].get_plugin_title().split(' - ')[0]
+                    text_item += (u"<span style= 'float: right; color:{0:};"
```
Comment (CAM-Gerlach) [inline spyder/widgets/fileswitcher.py]: Remove the extra space after ``span style= ``; add space after ``color:`` for style consistency.

File: spyder/widgets/fileswitcher.py [hunk 2]
```diff
@@ -669,17 +670,24 @@ def setup_file_list(self, filter_text, current_path):
...
                 if max_width > self.list.width():
-                    text_item += (u" &nbsp; <span style='color:{0:};"
+                    text_item += (u" &nbsp; <span style='float: left; color:{0:};"
                                   "font-size:{1:}pt'>{2:}"
                                   "</span>").format(self.PATH_FG_COLOR,
                                                     path_text_font_size,
                                                     short_paths[index])
                 else:
-                    text_item += (u" &nbsp; <span style='color:{0:};"
+                    text_item += (u" &nbsp; <span style='float: left; color:{0:};"
```
Comment (CAM-Gerlach) [inline spyder/widgets/fileswitcher.py]: Add space after ``color:`` for consistency.

File: spyder/widgets/fileswitcher.py [hunk 3]
```diff
@@ -669,17 +670,24 @@ def setup_file_list(self, filter_text, current_path):
...
                                   "font-size:{1:}pt'>{2:}"
                                   "</span>").format(self.PATH_FG_COLOR,
                                                     path_text_font_size,
                                                     paths[index])
+                    title = self.widgets[index][1].get_plugin_title().split(' - ')[0]
+                    text_item += (u"<span style= 'float: right; color:{0:};"
+                                  "font-size:{1:}pt'>{2:}"
```
Comment (CAM-Gerlach) [inline spyder/widgets/fileswitcher.py]: Add space after ``font-size:`` for consistency.

File: spyder/widgets/fileswitcher.py [hunk 4]
```diff
@@ -696,9 +704,8 @@ def setup_file_list(self, filter_text, current_path):
                 title = self.widgets[index][1].get_plugin_title().split(' - ')
                 if plugin != title[0]:
                     plugin = title[0]
-                    text += ("<br><big style='color:{0:}'>"
-                             "<b>{1:}</b></big><br>").format(ima.MAIN_FG_COLOR,
-                                                             plugin)
+                    text += ("<span style='color:{0:}'>{1:}"
```
Comment (CAM-Gerlach) [inline spyder/widgets/fileswitcher.py]: Shouldn't this be ``u"..."`` like the rest? Also, add space after ``color:`` for style consistency.

**Comment by goanpeca** [comment]
@juanis2112 and @dalthviz thanks for taking a look at this. I suggest we continue the integration of the new switcher (https://github.com/spyder-ide/spyder/pull/9133) which already makes this functionality much simpler, generic and decoupled from the actual process (of selecting files in this case)

**Comment by ccordoba12** [comment]
That's not going to be ready for beta2, so let's allow Juanita to finish this one.

**Comment by goanpeca** [comment]
(@ccordoba12 feels like duplicated effort.)

@juanis2112 this is how I accomplished it https://github.com/spyder-ide/spyder/pull/9133/files#diff-48bee778d0036aea166212d1e04c5ad0R195

**Comment by ccordoba12** [comment]
> feels like duplicated effort.

I know, but @juanis2112 has worked hard to implement this, so I'd like her fix to be part of beta2.

**Pull request merged**